game_eggs/eft/README.md

```diff
@@ -0,0 +1,51 @@
+# Escape from Tarkov
+
+# Singleplayer Tarkov Server
+
+## After server installation
+
+Run the server once to generate the necessary config files for the coop mod, then edit `coopConfig.json` in `/home/container/user/mods/SITCoop/config` with your external IPv4.
```

Comment:
This is not somthing generaly done. If that file is needed then you have yo generate it on install and set its value with the parser.

Reply:
The file doesn't seem really necessary for connection. I was able to connect from another machine without changing the config, so I'm not sure what it does.
It's included here because the [documentation for the coop mod](https://github.com/stayintarkov/SIT.Aki-Server-Mod#how-do-i-set-up-this-mod) calls for it.

Reply:
A disclaimer, before everything: I have no prior experience with TypeScript, everything here is the result of me staring at the code and looking up documentations.

I took a look at the SIT mod repo @ 1.5.1, it seems like the `externalIP` [config](https://github.com/stayintarkov/SIT.Aki-Server-Mod/blob/1.5.1/src/Overrides/LauncherControllerOverride.ts#L33) is only used to [store](https://github.com/stayintarkov/SIT.Aki-Server-Mod/blob/1.5.1/src/Overrides/GameControllerOverride.ts#L26) and [retrieve](https://github.com/stayintarkov/SIT.Aki-Server-Mod/blob/1.5.1/src/Overrides/GameControllerOverride.ts#L31) game session configs, so it shouldn't matter as long as it stays consistent. 
Plus, the [latest code](https://github.com/stayintarkov/SIT.Aki-Server-Mod/blob/master/src/CoopConfig.ts) doesn't have the field anymore, so it's probably in the process of being phased out.

Reply:
No. That value does need to be changed to an external ip address otherwise it won't be able to be connected too.